RELEASE_NOTES.md

```diff
@@ -13,6 +13,7 @@
 
 ## Fixed
 - For drop-in with sessions, error dialogs will no longer display user unfriendly messages.
+- Overriding some XML styles no longer causes a build error.
```

Comment:
It only happened for these styles if you'd omit the `parent`. Should we express more that it's kind of an edge case?

Reply:
I see your point but I can't find a way to clarify it, any suggestions?

Reply:
Maybe something like:
Overriding XML styles in some cases could cause a build error. This is now fixed.